gcp_variant_transforms/libs/variant_merge/move_to_calls_strategy.py

```diff
@@ -60,6 +60,39 @@ def __init__(self, info_keys_to_move_to_calls_regex, copy_quality_to_calls,
     self._copy_quality_to_calls = copy_quality_to_calls
     self._copy_filter_to_calls = copy_filter_to_calls
 
+  def move_data_to_calls(self, variant):
+    """Moves filters, calls, and info items to the variant's calls based on the
+    strategy's initialization parameters.
+
+    Args:
+      variant (:class: `~vcfio.Variant`): The variant whose filters, quality,
```

Comment:
nit: extra space after :class: ... also, I'd remove the :class: notation from here as we dont use that notation for anything other than beam_io (it's better to be consistent for now until we got back and fix everything)....so just `Variant` should be good.

Reply:
Done.

Reply:
Was this file not updated?

Reply:
Sorry, looks like I fixed it everywhere but here... Done now. :)